default.nix

```diff
@@ -39,7 +39,11 @@ with holonix.pkgs;
     ''
     ];
 
-  buildInputs = [ holonix.pkgs.libiconv ]
+  # Needed so bindgen can find libclang.so
+  LIBCLANG_PATH="${holonix.pkgs.llvmPackages.libclang}/lib";
+  hardeningDisable = [ "fortify" ];
```

Comment:
some of this needs to move upstream into holonix or it breaks the shell for other consumers